PyPDF2/filters.py

```diff
@@ -564,7 +564,13 @@ def _xobj_to_image(x_object_obj: Dict[str, Any]) -> Tuple[Optional[str], bytes]:
 
     :return: Tuple[file extension, bytes]
     """
-    from PIL import Image
+    try:
+        from PIL import Image
+    except ImportError:
+        raise ImportError(
+            "pillow is required to do image extraction. "
+            "It can be installed via 'pip install PyPDF2[image]'"
+        )
```

Comment:
If wanted code coverage here, could add a test with `monkeypatch.setitem` modifying `sys.modules` to remove it from the environment. See https://stackoverflow.com/a/28361013/4616655 for an example.

Reply:
That's an awesome idea! Thanks for sharing it - I've written the test!